Update/kage_012.txt

```diff
@@ -587,13 +587,13 @@ void main()
 //rもちろん、それはひょっとしたら、とんでもない自惚れなのかもしれない。…だけど、それでもまだ友達として『足りない』のだとしたら、いったい私に何が欠けていたんだろう？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "もちろん、それはひょっとしたら、とんでもない自惚れなのかもしれない。…だけど、それでもまだ友達として『足りない』のだとしたら、いったい私に何が欠けていたんだろう？",
-			NULL, "Of course, it might be ridiculously conceited of me to think it would work. ...Still, how on earth was I so <i>lacking</i> as a friend that I missed that?", Line_Normal);
+			NULL, "Of course, perhaps I was being ridiculously conceited to think that. ...Still, if I was <i>lacking</i> something as a friend, then what in the world was it?", Line_Normal);
```

Comment:
```suggestion
			NULL, "Of course, it might be ridiculously conceited of me to think that. ...Still, if I was <i>lacking</i> something as a friend, then what in the world was it?", Line_Normal);
```